resources/scss/bootstrap/_pagination.scss

```diff
@@ -1,9 +1,14 @@
 .pagination {
```

Comment:
Arg - I need to be more careful lol - NEVER modify bootstrap files. Always add an override in theme/ (usually components) and same for variables. Only update the theme variables. This way we can drop a new BS in and not worry about it.